release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -353,6 +353,42 @@ In {product-title} {product-version} and later, you can edit the baseboard manag
 [id="ocp-4-16-monitoring_{context}"]
 === Monitoring
 
+The in-cluster monitoring stack for this release includes the following new and modified features.
+
+[id="ocp-4-16-monitoring-updates-to-monitoring-stack-components-and-dependencies"]
+==== Updates to monitoring stack components and dependencies
+This release includes the following version updates for in-cluster monitoring stack components and dependencies:
+
+* kube-state-metrics to 2.12.0
+* Metrics Server to 0.7.1
+* node-exporter to 1.8.0
+* Prometheus to 2.52.0
+* Prometheus Operator to 0.73.2
+* Thanos to 0.35.0
+
+[id="ocp-4-16-monitoring-changes-to-alerting-rules"]
+==== Changes to alerting rules
```

Comment:
Hello @simonpasquier I remember you mentioned that the team creates a Jira for this. Is the Jira created somewhere? I could not find it and I wanted to update this section.
Thank you!

Reply:
https://issues.redhat.com/browse/MON-3892, I'll try to complete it ASAP.

Reply:
Oh thank you very much, that will be super helpful :)

Reply:
summarized the 4.16 alerting rules changes, see from https://issues.redhat.com/browse/MON-3892?focusedId=24948847&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-24948847, we can compare with the result from developers

Reply:
Thank you @juzhao and @simonpasquier I added the changes to the PR :+1: